.eslintrc.js

```diff
@@ -7,21 +7,16 @@ module.exports = {
     ecmaVersion: 2018,
     sourceType: 'module',
     ecmaFeatures: {
-      legacyDecorators: true
-    }
+      legacyDecorators: true,
+    },
   },
-  plugins: [
-    'ember'
-  ],
-  extends: [
-    'eslint:recommended',
-    'plugin:ember/recommended'
-  ],
+  plugins: ['ember'],
+  extends: ['eslint:recommended', 'plugin:ember/recommended', 'prettier'],
```

Comment:
[eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) will turn off some ESLint rules. As a result, some of our code conventions that were enforced by ESLint will not be enforced anymore. We could fix that by using Prettier as a linter but that will require to run `prettier --write` for all our .js files.

Reply:
I'll dig into this.